datatypes/src/collections/batch_builder.rs

```diff
@@ -144,6 +144,7 @@ impl RawFeatureCollectionBuilder {
         self.types.clone()
     }
 
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
why is this needed? It says
> Checks for private functions that only return Ok or Some.

 but this is not a private function

Reply:
The description maybe not be accurate but Clippy warns if I remove it.